packages/server/src/ui/entry.jsx

```diff
@@ -7,6 +7,11 @@
 import {h, render} from 'preact';
 import {App} from './app.jsx';
 
+// Fontsource for fonts/icons instead of google cdn
+import "@fontsource/roboto/400.css";
```

Comment:
missing roboto mono?

Reply:
Somehow I convinced myself that it wasn't included in the server public.html, but you're right- fixed.